libs/generic-view/feature/src/lib/setup-component.tsx

```diff
@@ -163,6 +169,38 @@ export const setupComponent = <P extends object>(
   }
 }
 
+const processFormFields = (
+  field: DataProviderExtendedField,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any
```

Comment:
Maybe replace `any` with `unknown` and use union type checks (e.g., `is List`, `is Boolean`) for safer type checking?

Reply:
Fixed in [`c8028ba` (#2076)](https://github.com/mudita/mudita-center/pull/2076/commits/c8028ba6b8efa5786ed2b360909d7f6c983e4cea)